src/styles/global.css

```
@@ -7,40 +12,62 @@
+@theme {
+  --color-background: #040406;
+  --color-card: #0a0c0f;
+  --color-elevated: #0c0e11;
+  --color-terminal: #12141a;
+  --color-border-default: #1a1d24;
+  --color-text-primary: #f0f1f2;
+  --color-text-secondary: #8c94a1;
+  --color-accent-mint: #66d9a5;
+  --color-text-light: #f5f7fa;
+  --color-text-accent: #66d9a5;
+  --color-accent-mint: #66d9a5;
```

Stylelint (17.10.0)

[error] Duplicate custom property "--color-accent-mint" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)